man/ukify.xml

```diff
@@ -112,6 +112,22 @@
 
       <para>The output files must not exist.</para>
     </refsect2>
+
+    <refsect2>
+      <title><command>inspect</command></title>
+
+      <para>Display information about the given PE, listing all known sections and
+      print the content of each (if text readable) together with displaying its size and
```

Comment:
There is a tense mismatch here: "listing" (present continuous) and "print" (present simple).